app/_src/production/secure-deployment/api-server-auth.md

```diff
@@ -301,6 +301,95 @@ kumactl delete global-secret admin-user-token
    {% endtab %}
    {% endtabs %}
 
+### Offline token issuing
```

Comment:
Shouldn't this whole thing be gated with

```
{% if_version gte:2.2.x %}
```

?

Reply:
yes!